### PR TITLE
Playwright Installation

### LINKED ISSUE #6598: Install Playwright (cypress dead)
On latest PR cypress tests are failing

```log
Process returned exit code 1
Launched as "pnpm start:static & pnpm wait-on http://localhost:9090 && MATRIX=4 node tests/cypress/e2e-run-tests.js"
Launched at "/__w/nft-gallery/nft-gallery"
Attempts exhausted, made 3 attempts :
Attempt #1 started at : 08:04:47 GMT+0000 (Coordinated Universal Time)
Attempt #2 started at : 08:05:18 GMT+0000 (Coordinated Universal Time)
Attempt #3 started at : 08:05:32 GMT+0000 (Coordinated Universal Time)
```
### Task
- migrate to [playwright](https://playwright.dev/) 

### ref
- https://playwright.dev/docs/intro
- https://www.browserstack.com/guide/playwright-vs-cypress
- https://apiumhub.com/tech-blog-barcelona/playwright-vs-cypress/
- https://betterprogramming.pub/choosing-playwright-over-cypress-deaf060e4a09

### PR DESCRIPTION
**Pull request by prury**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

Thanks for all the support, next step: rewriting old cypress tests

Who tests the tests added from the testers automated tests? 🐛 

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #6598 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

@stephenjason89 and @preschian did all the heavy lifting

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

image

### <samp>🤖 Generated by Copilot at 41de705</samp>

This pull request introduces Playwright tests for the NFT gallery project. It adds the `@playwright/test` package and its configuration, a GitHub Actions workflow to run the tests, and a basic test case for the Create Page. It also updates some dependencies and excludes the Playwright tests from Vitest.

### <samp>🤖 Generated by Copilot at 41de705</samp>

> _To test the NFT gallery site_
> _We added Playwright tests to write_
> _With `@playwright/test`_
> _And config file set_
> _We run them on GitHub with `workflow.yml`_

### CODE REVIEW

**Comment by stephenjason89** [comment]
Hello @prury, I believe the problem is with the playwright local webServer

so first we don't need this anymore
playwright.config.js
image

second
playwright.config.js
image
We need to point it on our dev server

third to execute the test we need to run our local server
pnpm dev

finally run test ❤️ 
pnpm exec playwright test
npx playwright show-report

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+
+    - name: Install dependencies
+      uses: ./.github/actions/pnpm-install
```
Comment (preschian) [inline .github/workflows/playwright.yml:18]: 1. add `pnpm generate` after `pnpm-install` 

https://github.com/kodadot/nft-gallery/blob/872e7fdcc8cdcac1c2558a0c0c4f0279e59f329a/.github/workflows/build.yml#L30-L31

File: playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    // },
+    // {
+    //   name: 'Google Chrome',
+    //   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+    // },
+  ],
+
+  /* Run your local dev server before starting the tests */
+  webServer: {
+    command: 'pnpm run dev',
```
Comment (preschian) [inline playwright.config.ts:73]: 2. change it to `process.env.CI ? 'pnpm start:static' : 'pnpm run dev'`. on CI we test against production build

```suggestion
    command: process.env.CI ? 'pnpm start:static' : 'pnpm run dev',
```

**Review by preschian — COMMENTED** [review]
3. exlude `test/e2e` folder on vitest

https://github.com/kodadot/nft-gallery/blob/872e7fdcc8cdcac1c2558a0c0c4f0279e59f329a/vitest.config.js#L10-L14

**Review by preschian — APPROVED** [review]
cool, next PR,
time to migrate and adjust cypress test to the playwright

**Comment by prury** [comment]
> cool, next PR, time to migrate and adjust cypress test to the playwright

yes! Thank you so much @preschian

**Review by roiLeo — APPROVED** [review]
✅ all green

**Comment by vikiival** [comment]
Amazing ❤️ ❤️

**Comment by yangwao** [comment]
What is current Playwright test coverage?

**Comment by prury** [comment]
> What is current Playwright test coverage?

basically test if the page exists, can you open issue for me to work on test old cypress tests conversion?

**Pull request merged**